source/local/local.go

```diff
@@ -272,8 +297,20 @@ func getFeaturesFromFiles() (map[string]string, error) {
 			continue
 		}
 
+		// Parse comments
+		comments := parseComments(lines, fileName)
```

Comment:
We' re already parsing line by line here. No need/sense to parse all comment lines per each line. I suggest to simplify this a lot. E.g. above if `HasPrefix(line, "#")` then do parseComment() for that (one) line.

Reply:
You mean in the `parseFeatures()` function we check for each line if it's a comment we parse it?

Reply:
Yeah, that's what I was thinking. In any case it doesn't make any sense to for every line to re-parse the whole file for comments (plus we lose the trimming of whitespace we did earlier).